cli/src/main/scala-2.12/coursier/cli/Helper.scala

```diff
@@ -291,10 +291,27 @@ class Helper(
     .flatMap {
       case (dep, extraParams) =>
         extraParams.get("url").map { url =>
-          dep.moduleVersion -> (new URL(URLDecoder.decode(url, "UTF-8")), true)
+          dep.moduleVersion -> (convertStringToUrl(url), true)
         }
     }.toMap
 
+  private def convertStringToUrl(url: String): URL = {
```

Comment:
might need some docstring for this. iiuc this defines the behavior to obtain URLConnection for anything with `local://`